platforms/hyperledger-fabric/charts/catools/templates/deployment.yaml

```diff
@@ -28,22 +28,676 @@ spec:
       labels:
         name: {{ .Values.metadata.name }}
     spec:
-      containers:
-        - name: {{ .Values.metadata.name }}
-          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
-          imagePullPolicy: {{ .Values.image.pullPolicy }}
-          command: ["sh", "-c", "/bin/bash"]
-          stdin: true
-          tty: true
-          volumeMounts:
-          - mountPath: /root/ca-tools
-            name: ca-tools-pv
-          - mountPath: /crypto-config
-            name: ca-tools-crypto-pv
+      serviceAccountName: {{ $.Values.vault.serviceaccountname }}
```

Comment:
@sownak I believe this should be a deployment only.
Because, the approach followed in this was to create an extra main container in the ca-tools pod which will wait for the cli to be started and then generate the crypto material and store in the vault.
But if we make it as a job, then the ca-cli wont be available for later use.

Reply:
I see a vault copy container alongside the container that generate crypto materials for an organization. The vault copy container is made to tail on null device, making it wait infinitely. Is ca tools expected to be always running?

Reply:
What's the scenario where `catools` will help? By moving everything out of Ansible, we're not dependent on the state anymore right?

Reply:
The catools is also used to run [manage user certificates playbook](https://github.com/hyperledger-labs/blockchain-automation-framework/blob/develop/platforms/hyperledger-fabric/configuration/manage-user-certificate.yaml) and there will be changes to it as well if we make this as a job and not as a deployment.